doc/classes/Input.xml

```diff
@@ -312,6 +312,113 @@
+				[param strengths] array must have exactly 10 elements, and each of them can take values from 0 to 8 (inclusive).
+				This method returns [code]true[/code] if the effect request was sent successfully, otherwise this method returns [code]false[/code].
+				[b]Note:[/b] This feature is only supported on Windows, Linux and macOS.
+			</description>
+		</method>
+		<method name="joy_adaptive_triggers_multi_vibration">
+			<return type="bool" />
+			<param index="0" name="device" type="int" />
+			<param index="1" name="axis" type="int" enum="JoyAxis" />
+			<param index="2" name="frequency" type="int" />
+			<param index="3" name="amplitudes" type="int[]" />
+			<description>
+				Starts "Multi Vibration" adaptive triggers effect on the specified joypad. This method will fail if the joypad is not DualSense.
+				Axis should be [constant JOY_AXIS_TRIGGER_LEFT] for the left trigger, [constant JOY_AXIS_TRIGGER_RIGHT] for the right trigger, or [code]JOY_AXIS_TRIGGER_LEFT+JOY_AXIS_TRIGGER_RIGHT[/code] for both triggers at once.
+				[param frequency] parameter can take values from 0 to 255 (inclusive).
+				[param amplitudes] array must have exactly 10 elements, and each of them can take values from 0 to 8 (inclusive).
+				This method returns [code]true[/code] if the effect request was sent successfully, otherwise this method returns [code]false[/code].
+				[b]Note:[/b] This feature is only supported on Windows, Linux and macOS.
+			</description>
+		</method>
+		<method name="joy_adaptive_triggers_off">
+			<return type="bool" />
+			<param index="0" name="device" type="int" />
+			<param index="1" name="axis" type="int" enum="JoyAxis" />
+			<description>
+				Stops adaptive triggers effects on the specified joypad. This method will fail if the joypad is not DualSense.
+				Axis should be [constant JOY_AXIS_TRIGGER_LEFT] for the left trigger, [constant JOY_AXIS_TRIGGER_RIGHT] for the right trigger, or [code]JOY_AXIS_TRIGGER_LEFT+JOY_AXIS_TRIGGER_RIGHT[/code] for both triggers at once.
+				This method returns [code]true[/code] if the effect request was sent successfully, otherwise this method returns [code]false[/code].
+				[b]Note:[/b] This feature is only supported on Windows, Linux and macOS.
+			</description>
+		</method>
+		<method name="joy_adaptive_triggers_slope_feedback">
+			<return type="bool" />
+			<param index="0" name="device" type="int" />
+			<param index="1" name="axis" type="int" enum="JoyAxis" />
+			<param index="2" name="start_position" type="int" />
+			<param index="3" name="end_position" type="int" />
+			<param index="4" name="start_strength" type="int" />
+			<param index="5" name="end_strength" type="int" />
+			<description>
+				Starts "Slope Feedback" adaptive triggers effect on the specified joypad. This method will fail if the joypad is not DualSense.
```

Comment:
```suggestion
				Starts the "Slope Feedback" adaptive trigger effect on the specified joypad. This method will fail if the joypad does not have adaptive triggers.
				The trigger specified by [param axis] will start providing the resistance [param start_strength] when pressed in to [param start_position]. The resistance will increase as the trigger is pressed further; it will reach [param end_strength] once the trigger is pressed to [param end_position], and will maintain that resistance past that point.
```